x/arkeo/types/message_claim_contract_income.go

```diff
@@ -66,26 +65,21 @@ func (msg *MsgClaimContractIncome) GetSignBytes() []byte {
 	return sdk.MustSortJSON(bz)
 }
 
+func (msg *MsgClaimContractIncome) GetBytesToSign() []byte {
+	return GetBytesToSign(msg.ContractId, msg.Spender, msg.Height, msg.Nonce)
+}
+
+func GetBytesToSign(contractId uint64, spender common.PubKey, height, nonce int64) []byte {
```

Comment:
I fail to see why move this one line to a seperate func here

Reply:
After many refactorings it is unused.  It originally was there to avoid having to have the full message struct in order to generate the needed bytes.  At this point, seems like we dont need it, but doesn't really hurt to have it either.  If you feel strongly about removing it, I can.